client/src/containers/private-route/index.tsx

```diff
@@ -13,8 +13,8 @@ interface IPrivateRouteProps extends RouteProps {
 
 const PrivateRoute = (props: IPrivateRouteProps) => {
 	const { component: Component, ...rest } = props;
-	// const { user } = useAppSelector((state) => state.auth.userData);
-	const isAuthorized = true;
+	const { user } = useAppSelector((state) => state.auth.userData);
+	const isAuthorized = Boolean(user);
```

Comment:
oh, that was why not working, ahaha
and I thought something serious went wrong :D